### PR TITLE
Kam user secret mapping

### PR DESCRIPTION
**Pull request by amitkrout**
Injecting a github token to crate/remove the repo via github cli in prow CI test infra.

Reference doc - https://github.com/openshift/release/tree/master/core-services/secret-mirroring#self-managed-secrets

Steps i followed:
```
$ oc login https://api.ci.openshift.org --token=******************************
Logged into "https://api.ci.openshift.org:443" as "amitkrout" using the token provided.

[...]

$ oc new-project kam
Now using project "kam" on server "https://api.ci.openshift.org:443".
[...]

$ oc create secret --namespace kam generic kam-user-secret --from-file secret.txt
secret/kam-user-secret created
```
NOTE: I created a regular secret.txt file and put the token that i created for kam github user. Find details here at -  https://issues.redhat.com/browse/GITOPS-333?focusedCommentId=15333273&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15333273

### CODE REVIEW

**Comment by amitkrout** [comment]
ping @wtam2018 @bigkevmcd

**Comment by wtam2018** [comment]
@amitkrout 

Was the secret create in `kam` or `odo` namspace?     You have this command in the description. 
```
$ oc create secret --namespace odo generic kam-user-secret --from-file secret.txt
secret/kam-user-secret created
```

File: core-services/secret-mirroring/_mapping.yaml [hunk 1]
```diff
@@ -240,3 +240,15 @@ secrets:
...
+    namespace: kam
+    name: kam-user-secret
+  to:
+    namespace: ci
+    name: kam-user-secret
+- from:
+    namespace: kam
+    name: kam-user-secret
+  to:
+    namespace: ci-stg
```
Comment (alvaroaleman) [inline core-services/secret-mirroring/_mapping.yaml:253]: Please do not put anything into `ci-stg`, its not used anymore

**Comment by amitkrout** [comment]
> @amitkrout
> 
> Was the secret create in `kam` or `odo` namspace? You have this command in the description.
> 
> ```
> $ oc create secret --namespace odo generic kam-user-secret --from-file secret.txt
> secret/kam-user-secret created
> ```

Ahh, my mistake. Hope nothing should break for odo 
```
$ oc create secret --namespace kam generic kam-user-secret --from-file secret.txt
secret/kam-user-secret created

$ oc get secrets --namespace kamNAME                       TYPE                                  DATA   AGE
builder-dockercfg-9zmql    kubernetes.io/dockercfg               1      7m
builder-token-4bxpq        kubernetes.io/service-account-token   4      7m
builder-token-6gm8z        kubernetes.io/service-account-token   4      7m
default-dockercfg-l7stj    kubernetes.io/dockercfg               1      7m
default-token-2pv89        kubernetes.io/service-account-token   4      7m
default-token-h947w        kubernetes.io/service-account-token   4      7m
deployer-dockercfg-sw9cr   kubernetes.io/dockercfg               1      7m
deployer-token-8mfbn       kubernetes.io/service-account-token   4      7m
deployer-token-gvmcs       kubernetes.io/service-account-token   4      7m
kam-user-secret            Opaque                                1      7s 
```
Updated. @wtam2018 thanks for pointing it.

**Review by alvaroaleman — APPROVED** [review]

File: core-services/secret-mirroring/_mapping.yaml (hunk 1 above)
Comment (amitkrout) [reply core-services/secret-mirroring/_mapping.yaml]: Done

@alvaroaleman I have created a pr https://github.com/openshift/release/pull/13039 to update the README.md. PTAL

**Pull request merged**